sdk/communication/azure-communication-networktraversal/README.md

```diff
@@ -1,230 +1,10 @@
 # Azure Communication Network Traversal Package client library for Java
 
-Azure Communication Network Traversal is managing TURN credentials for Azure Communication Services.
-
-It will provide TURN credentials to a user.
-
-[Source code](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication) | [API reference documentation](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication)
+This service is shutting down on 03/31/2024 and there is no replacement, please read our [retirement notice](https://azure.microsoft.com/en-us/updates/retirement-notice-azure-communication-services-network-traversal-turn-public-preview-is-retiring/) for more details.
 
 ## Getting started
```

Comment:
remove all the empty headings

Reply:
The pipeline will compain about the heading missing after removing them